### DISCUSSION TITLE
Auto-Filter by Creator or Modifier

### DISCUSSION THREAD

**Discussion by ferrarirosso** [microsoft-search/pnp-modern-search#3166] [Q&A]
Is there a way to filter by default the "created by / modified by"-columns setting the current user as the filter value ?
Like it is possible with the [Me]-Keywords for Listview [Filter to modify Sharepoint View](https://support.microsoft.com/en-us/office/use-filtering-to-modify-a-sharepoint-view-3d8efc52-0808-4731-8f9b-3dfaeacea3d4)

![Screenshot 2023-08-01 at 11 08 32](https://github.com/microsoft-search/pnp-modern-search/assets/5904018/d1a1bf3c-b018-444e-8fb2-21824b1a6cba)

Bonus : if the filter could be auto-set by could be removed by the user 

If it's not possible, what would the best path to implement this ?

Thanks !

**Comment by kasperbolarsen** [comment]
I think you could use the User property in the Url , would that fit your purpose?

https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/

**Reply by ferrarirosso** [reply]
@kasperbolarsen Many thanks ! I was not aware of these possibilities, will test it
